python/fbprophet/forecaster.py

```diff
@@ -1474,3 +1505,57 @@ def copy(self, cutoff=None):
             DeprecationWarning,
         )
         return prophet_copy(m=self, cutoff=cutoff)
+
+    def make_holidays(self, year_list, country_list):
+        """Make dataframe of holidays for given years and countries
+
+        Parameters
+        ----------
+        cls: Prophet class
+        year_list: a list of years
+        country_list: a list of countries
+
+        Returns
+        -------
+        Dataframe with 'ds' and 'holiday', which can directly feed
+        to 'holidays' params in Prophet
+        """
+
+        if isinstance(country_list, str):
```

Comment:
I think this can be simplified by doing
```
if isinstance(country_list, str):
    country_list = [country_list]
```
and then cutting directly to the code in L1539